flake-modules/dev/default.nix

```diff
@@ -20,6 +20,7 @@
         projectRootFile = "flake.nix";
 
         programs = {
+          black.enable = true;
```

Comment:
Can we use `ruff` here ?

Reply:
https://flake.parts/options/treefmt-nix#opt-perSystem.treefmt.programs.ruff.format

Reply:
And even for linting as well: https://flake.parts/options/treefmt-nix#opt-perSystem.treefmt.programs.ruff.check

(not sure we need that though)

Reply:
Done, didn't change the output format though.